site/pages/technology/style.css

```diff
@@ -0,0 +1,94 @@
+@value badgerWhite, badgerRedOnWhite, badgerRedOnBlack, badgerBlack from "../../css/_colors.css";
+@value mediumScreen, largeScreen, contentMaxWidth from "../../css/_sizes.css";
+
+.technologyPage {
+  padding: 0px 20px;
+  background-color: badgerWhite;
+}
+
+.intro {
+  max-width: contentMaxWidth;
+  margin: auto;
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+.leftContent, .rightContent {
+  padding-bottom: 50px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.rightContent {
+  padding-top: 30px;
+}
+
+@media mediumScreen {
+  .intro {
+    flex-direction: row;
+    padding: 50px 0px 50px 0px;
+  }
+
+  .leftContent, .rightContent {
+    flex: 1 50%;
+    align-items: center;
+  }
+
+  .leftContent {
+    padding: 40px 10px 50px 0px;
+    justify-content: flex-start;
+  }
+
+  .rightContent {
+    justify-content: flex-end;
+  }
+
+  .innerContainer {
+    max-width: 25em;
+  }
+}
+
+@media largeScreen {
+  .rightContent > .innerContainer {
+    max-width: 500px;
+  }
+}
+
+
+.mainHeader {
+  composes: fontL from "../../css/typography/_fonts.css";
+  composes: serif from "../../css/typography/_fonts.css";
+  color: badgerBlack;
+}
+
+.description {
+  composes: fontS from "../../css/typography/_fonts.css";
+  composes: serif from "../../css/typography/_fonts.css";
+  color: badgerBlack;
+  line-height: 1.21;
+}
+
+.latestRoundTableLink {
+  font-family: 'ProximaNova-Bold', sans-serif;
+  border-bottom: 2px solid #E5051A;
+  text-decoration: none;
+  color: #fc1d42;
```

Comment:
Can we use badgerRedOnWhite here please @Xanndarr? It should pass accessibility tests then.